.github/workflows/master-e2e.yaml

```diff
@@ -400,7 +407,14 @@ jobs:
       - name: Uninstall Elemental Operator
         # Don't test Operator uninstall if we want to keep the runner for debugging purposes
         if: inputs.destroy_runner == true && inputs.test_type == 'cli'
-        run: cd tests && make e2e-uninstall-operator
+        run: |
+          OPERATOR_VERSION=$(echo ${{ steps.component.outputs.operator_version }} \
+                             | awk -F ':' '{print substr($2,0,3)}')
```

Comment:
It took me a while to realize that in the stable release the version has not the "v" prepended and so here it becomes "1.0"!